dali/operators/reader/numpy_reader_op.cc

```diff
@@ -14,20 +14,22 @@
 
 #include <string>
 
+#include "dali/core/backend_tags.h"
 #include "dali/kernels/slice/slice_cpu.h"
 #include "dali/kernels/slice/slice_flip_normalize_permute_pad_cpu.h"
 #include "dali/kernels/transpose/transpose.h"
 #include "dali/core/static_switch.h"
 #include "dali/operators/reader/numpy_reader_op.h"
+#include "dali/pipeline/data/backend.h"
 
 namespace dali {
 
-static void CopyHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &input,
+static void CopyHelper(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input,
```

Comment:
How about a meaningful name?
```suggestion
static void ParallelCopy(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input,
```

Reply:
Does it mean that all the functions that also utilize the thread pool require a Parallel prefix instead of Helper suffix?

Reply:
I do strongly believe that a function's name should explain what the function does (within reason). "Helper" adds very little if anything. Having taken another look at this function, it should probably be "ScheduleCopy", because the copy operations are scheduled and there's no `wait` at the end.